UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Controllers/SubscriptionControllerTests.cs

```diff
@@ -22,15 +22,34 @@ public void Setup()
             _fakeHttpContextAccessor = A.Fake<IHttpContextAccessor>();
             _fakeLogger = A.Fake<ILogger<SubscriptionController>>();
             A.CallTo(() => _fakeHttpContextAccessor.HttpContext).Returns(new DefaultHttpContext());
+
             _controller = new SubscriptionController(_fakeHttpContextAccessor, _fakeLogger);
         }
 
         [Test]
         public async Task TestSubscription()
```

Comment:
Appreciate it's not changed in this PR, but can we ensure the tests are named using the When_Then naming convention  - thanks.

So this may be "WhenPostValidPayload_ThenRecieveSuccessfulResponse"

Reply:
Thanks @Shirin14926 for doing this 😃

Reply:
Cheers!!!